docs/src/components/productX/XDataGrid.tsx

```diff
@@ -206,13 +210,33 @@ export default function XDataGrid() {
                   display: 'none',
                 },
               },
+              overflow: 'hidden',
             }}
           >
             <Box sx={{ position: 'relative' }}>
               <Box sx={{ position: 'relative', zIndex: 1 }}>
                 <HighlightedCode component={MarkdownElement} code={code} language="jsx" />
               </Box>
               {demo && <FlashCode startLine={startLine[demo]} sx={{ mx: -2 }} />}
+              <StylingInfo
+                appeared={demo === DEMOS[3]}
```

Comment:
We should add in the Filtering feature as well. In that case, the title would be: _"**Multi-column filtering is a paid feature!**_ and the description the same.